tests/Feature/UserTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\User as User;
+
+
+class UserTest extends TestCase
+{
+
+    private $baseURL ='api/v1/user/';
+    use RefreshDatabase;
+
+    
+   public function register()
+   {
+       return  $this->post($this->baseURL.'create',[
+        'name'=>'Test Doe',
+        'email'=>'test@email.com',
+        'password'=>'testpassword',
+    ]);
+   }
+
+
+    /** @test @return void */
+    public function new_user_can_register()
+    {
+        $response =$this->register();
+
+        $this->assertCount(1,User::all());
+        $response->assertStatus(201);
+   }
+
+
+   /** @test @return void */
+   public function email_is_unique()
+   {
+    $response =  $response =$this->register();
+
+    $response =   $response =$this->register();
+
+
+    $this->assertCount(1,User::all());
+    $response->assertJSON(['result'=>0]);
+ }
+
+    /** @test @return void */
+    public function user_can_login()
+    {
+        $this->register();
+        $this->assertCount(1,User::all());
+
+        $response = $this->post($this->baseURL.'login',[
+            'email'=>'test@email.com',
+            'password'=>'testpassword',
+        ]);
+        $response->assertJSON(['result'=>1]);
+        $response->assertOk();
+
+    }
+  /** @test @return void */
+  public function wrong_user_cannot_login()
+  {
+      $this->register();
+      $this->assertCount(1,User::all());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\App\User' in method 'wrong_user_cannot_login'.](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5510282)